python/ql/test/experimental/dataflow/tainttracking/defaultSanitizer/TestTaint.expected

```diff
@@ -1,5 +1,37 @@
-| test.py:16 | fail | const_eq_clears_taint | ts |
-| test.py:18 | ok   | const_eq_clears_taint | ts |
-| test.py:24 | fail | const_eq_clears_taint2 | ts |
-| test.py:29 | ok   | non_const_eq_preserves_taint | ts |
-| test.py:31 | ok   | non_const_eq_preserves_taint | ts |
+| test_logical.py:30 | fail | test_basic | s |
```

Comment:
Not all of these `fail`ures are indicated in the source code. It would be nice if they were (unless I'm misreading this output).

Reply:
Agreed. I did some work on using the inline test expectations to achieve this (and make sure the indications are up to date), but I didn't quite find a solution I was happy with yet. -- would you be OK with not having any until we find a good solution to this? :blush:

Reply:
Yeah, that's fine. I think we'll want to extend the inline test expectations library to fit our use case, so updating the tests can happen once that's been done.